tests/tools/cirdl/testcirdl.nim

```diff
@@ -2,10 +2,11 @@ import std/os
 import std/osproc
 import std/options
 import pkg/chronos
-import codex/contracts
-import ../../integration/marketplacesuite
+import pkg/codex/contracts
+import ../../asynctest
+import ../../contracts/deployment
 
-marketplacesuite "tools/cirdl":
+suite "tools/cirdl":
```

Comment:
The test starts circuit-downloader process, which tries to talk to an RPC endpoint at "ws://localhost8545".
Is this going to work without marketplacesuite?